scripts/creation-portal-migration/orgDetails.js

```diff
@@ -0,0 +1,140 @@
+require("dotenv").config({ path: "./../../.env" });
+const { createDBInstance } = require("./db/dbConfig");
+const { findAll } = require("./db");
+const { CONFIG } = require("./constant/config");
+const { compact, uniq } = require("lodash");
+const fs = require("fs");
+const { searchUser, getOpenSaberUserOrgId } = require("./api-list/user");
+
+/**
+ * To get the userData from creation portal
+ * @method
+ * @name getUserIds
+ **/
+const getUserIds = async () => {
```

Comment:
@jayesh9747 update the function signature, and wha will be the response, its return user details or just Id ?